connectors/jetty-connector/src/test/java/org/glassfish/jersey/jetty/connector/AsyncTest.java

```diff
@@ -110,29 +110,32 @@ public void handleTimeout(AsyncResponse asyncResponse) {
                 }
             });
             asyncResponse.setTimeout(1, TimeUnit.SECONDS);
-
-            new Thread(new Runnable() {
-
-                @Override
-                public void run() {
-                    String result = veryExpensiveOperation();
-                    asyncResponse.resume(result);
-                }
-
-                private String veryExpensiveOperation() {
-                    // very expensive operation that typically finishes within 1 second but can take up to 5 seconds,
-                    // simulated using sleep()
-                    try {
-                        Thread.sleep(5 * OPERATION_DURATION);
-                        return "DONE";
-                    } catch (InterruptedException e) {
-                        Thread.currentThread().interrupt();
-                        return "INTERRUPTED";
-                    } finally {
-                        LOGGER.info("Async long-running get with timeout finished on thread " + Thread.currentThread().getName());
-                    }
-                }
-            }).start();
+            asyncResponse.resume(Response.status(Response.Status.SERVICE_UNAVAILABLE)
+                    .entity("Operation time out.").build());
+
+//            new Thread(new Runnable() {
+//
+//                @Override
+//                public void run() {
+//                    String result = veryExpensiveOperation();
+//                    asyncResponse.resume(result);
+//                }
+//
+//                private String veryExpensiveOperation() {
+//                    // very expensive operation that typically finishes within 1 second but can take up to 5 seconds,
+//                    // simulated using sleep()
+//                    try {
+//                        Thread.sleep(5 * OPERATION_DURATION);
+//                        return "DONE";
+//                    } catch (InterruptedException e) {
+//                        Thread.currentThread().interrupt();
+//                        return "INTERRUPTED";
+//                    } finally {
+//                        LOGGER.info("Async long-running get with timeout finished on thread " +
+//            Thread.currentThread().getName());
+//                    }
+//                }
+//            }).start();
```

Comment:
If the above code is not important any more, you should remove it. Could you please provide some explanation why you think it should be gone?

Reply:
It's some time ago I did the original commenting out. This one probably should be un-commented out again. The reason I commented it out in the original commit is that it's a very fragile test. It would fail a lot of times even before any changes were made to the code.

So to make testing easier I commented it out, but it's unrelated to this PR.